### DISCUSSION TITLE
Turbo mode with 3rd party power supply

### DISCUSSION THREAD

**Discussion by andibo73** [seerge/g-helper#791] [Q&A]
Hello,
is the turbo mode in G-Helper the same as in Armory Crate? In other words, does it depend on the official power supply too or does it deliver the same performance even with a 3rd party adapter?
In AC the turbo mode can only be selected with the official Asus power supply.
My model is the GZ301VIC with a 130W power supply. Alternatively I'd like to use a 140W PD3.1 USB-C adapter.
Thanks in advance
Andreas

**Comment by seerge** [comment]
@andibo73 all modes are baked in bios, so yes - it's same as in Armoury. But keep in mind that it may behave differently on USB-C vs Barrel anyway (cause again this is hardcoded in bios)

**Reply by andibo73** [reply]
Thanks for the fast respond.
The official power supply is USB-C, too.
So if the modes are hardcoded in bios, it is the software that verifies the power supply, right?
And G-Helper lets you select turbo mode anyway. Even on battery.

**Reply by seerge** [reply]
Bios verifies power supply - yes :) App just sets mode (one of 3 predefined)
